crates/subspace-networking/src/bin/subspace-bootstrap-node/main.rs

```diff
@@ -155,6 +167,22 @@ async fn main() -> Result<(), Box<dyn Error>> {
                 })
                 .transpose()?;
 
+            let known_peers_registry_config = NetworkingParametersManagerConfig {
+                enable_known_peers_source: false,
```

Comment:
I'm not sure we need this parameter and to disable it at all. What I just thought about is that contents of k-buckets is not the same as known peers, so shouldn't we go through known peers and verify them periodically?

Reply:
We discussed this parameter offline yesterday. Previously, we treated the file as a storage that last longer than Kademlia memory, this is why we have two separate delays for address removal.

Reply:
Known peers still don't have to match k-buckets necessarily, so I'm still wondering why do we need this parameter and set to to `false`. It should work fine with it set to `true` too.